core/src/test/scala/com/spotify/featran/transformers/mdl/ThresholdFinderTest.scala

```diff
@@ -25,7 +25,7 @@ class ThresholdFinderTest extends AnyFlatSpec with Matchers {
     val finder =
       new ThresholdFinder(nLabels = 3, stoppingCriterion = 0, maxBins = 100, minBinWeight = 1)
 
-    val feature = Array(
```

Comment:
To avoid the warning in the usage bellow
```
[warn]  method copyArrayToImmutableIndexedSeq in class LowPriorityImplicits2 is deprecated (since 2.13.0): Implicit conversions from Array to immutable.IndexedSeq are implemented by copying; Use the more efficient non-copying ArraySeq.unsafeWrapArray or an explicit toIndexedSeq call
[warn]     val result = finder.findThresholds(feature)
```